samples/subsys/ipc/ipc_service/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ set(REMOTE_ZEPHYR_DIR ${CMAKE_CURRENT_BINARY_DIR}/ipc_service_remote-prefix/src/
 
 if("${BOARD}" STREQUAL "nrf5340dk_nrf5340_cpuapp")
   set(BOARD_REMOTE "nrf5340dk_nrf5340_cpunet")
+elseif("${BOARD}" STREQUAL "bl5340_dvk_cpuapp")
+  set(BOARD_REMOTE "bl5340_dvk_cpunet")
```

Comment:
I see you're not the one introducing this, but I find this to restrictive in general.

There will be more boards capable of supporting this sample.
Like all `nRF5340` based boards, and most likely also several lpcxpresso boards and some other boards.

We have several samples where there are specific requirements to the hw for running the sample, but we don't filter on boards in those samples, for example the `can/isotp` sample requires CAN bus support, and not all boards have that.

But seems to have been used in most samples in `subsys/ipc`.